app/Http/Controllers/Common/SystemManagerController.php

```diff
@@ -17,15 +19,30 @@ public function __construct()
 
     public function getSystemManagers()
     {
-        $accountManagers = User::where('role', 'admin')
+        $accountManagers = User::selectRaw("id, CONCAT(first_name, ' ', last_name, ' <', email, '>') AS display")
```

Comment:
Never query a table twice in the same method. Use where, whereIn, groupby and map